.travis.yml

```diff
@@ -1,14 +1,19 @@
 cache: yarn
-
 language: node_js
-node_js:
-  - "6.7"
 
-script:
-  - scripts/deploy_commit.sh
-  - yarn ci
-  - yarn danger
-  - scripts/deploy_apphub.sh
+matrix:
+  include:
+  - node_js: '6.7'
+    script:
+      - yarn ci
+      - yarn danger
+
+  # Deployment related tasks
+  - node_js: "node"
```

Comment:
Why is this version requirement different than the one above?

Reply:
You can't have the same env var for both

Reply:
Ah. Do we still do `nvm use` after the VM has started? I’d prefer it that the deployment runs on the same versions we decided to use locally.

Reply:
We could, that should work. 👍

Reply:
Yep that works - but THB, I'm not convinced that it's uploading when it should be - will take another look